src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Methodologies/MethodologyController.cs

```diff
@@ -21,53 +20,53 @@ public MethodologyController(IMethodologyService methodologyService)
             _methodologyService = methodologyService;
         }
 
-        [Produces("application/json")]
-        [HttpGet("api/topic/{topicId}/methodologies")]
-        public Task<ActionResult<List<MethodologyViewModel>>> GetTopicMethodologiesAsync([Required]Guid topicId)
-        {
-            return _methodologyService
-                .GetTopicMethodologiesAsync(topicId)
-                .HandleFailuresOrOk();
-        }
-        
         [Produces("application/json")]
         [HttpGet("api/methodologies")]
-        public Task<ActionResult<List<MethodologyViewModel>>> GetMethodologiesAsync()
+        public Task<ActionResult<List<MethodologyStatusViewModel>>> GetMethodologiesAsync()
         {
             return _methodologyService
                 .ListAsync()
                 .HandleFailuresOrOk();
         }
-        
+
         [Produces("application/json")]
-        [ProducesResponseType(typeof(MethodologyViewModel), 200)]
+        [ProducesResponseType(typeof(MethodologyTitleViewModel), 200)]
         [ProducesResponseType(typeof(ValidationProblemDetails), 400)]
         [HttpPost("api/methodologies")]
-        public Task<ActionResult<MethodologyViewModel>> CreateMethodologyAsync(
-            CreateMethodologyViewModel methodology)
+        public Task<ActionResult<MethodologyTitleViewModel>> CreateMethodologyAsync(
+            CreateMethodologyRequest methodology)
```

Comment:
I think we'll also need a PUT endpoint to be able to update the metholodgy details. Think this should just be title and scheduled publish date?

Reply:
That's been added now. The latest changes are

- Added endpoint to update a Methodology.
- Changed the create to return `MethodologySummaryViewModel` rather than `MethodologyTitleViewModel`.
- Removed the endpoint to get a Methodology status.

A future change needs to remove the dedicated endpoint to update a Methodology status once that's no longer used by the front end, and remove `UpdateMethodologyStatusRequest`.

`MethodologyStatusViewModel` is still in use by the view that shows all methodologies.